cmd/trcli/config/model.go

```diff
@@ -25,27 +25,27 @@ func (e Endpoint) RawParams() string {
 }
 
 type Runtime struct {
-	Type   string
-	Params interface{}
+	Type   string      `json:"type" yaml:"type"`
```

Comment:
this one is backward incompatible change, but in this case it's fine, let's extract this in separate PR.